core/trino-parser/src/main/java/io/trino/sql/parser/AstBuilder.java

```diff
@@ -729,16 +729,11 @@ public Node visitSetViewAuthorization(SqlBaseParser.SetViewAuthorizationContext
     @Override
     public Node visitSetMaterializedViewProperties(SqlBaseParser.SetMaterializedViewPropertiesContext context)
     {
-        List<Property> properties = ImmutableList.of();
```

Comment:
squash with relevant commit that introduces `visitSetMaterializedViewProperties`